.gitignore

```diff
@@ -25,3 +25,7 @@ yarn-error.log*
 # misc
 .DS_Store
 .idea
+
+# pnpn & npm lockfiles
+pnpm-lock.yaml
+package-lock.json
```

Comment:
I also added these two lines, so I don't have to use `--no-lockfile` flag every time.